src/test/kotlin/blackjack/controller/GameManagerTest.kt

```diff
@@ -15,29 +17,29 @@ import org.junit.jupiter.api.Test
 class GameManagerTest {
     @Test
     fun setUp() {
-        val participants = Participants.from(listOf("mina", "guri"))
+        val participants = Participants(DummyPlayerFactory(2).players)
         GameManager(participants).setUp()
 
-        OutputView.showCards(participants.getDealer())
-        OutputView.showAllPayersCards(participants)
+        OutputView.showCards(participants.dealer)
+        OutputView.showAllPayersCards(participants.players)
```

Comment:
Although I didn't leave this review in the previous PR, it would be great to have at least one assertion in tests instead of having UI result. This is because we don't actually see the whole test result logs whenever we run the tests. That means even if the output is wrong, the test will still pass and we won't notice it. In this case, I don't think this test is necessary, with reasons I've written in the previous comment.